environments/drupal-8.sh

```diff
@@ -15,6 +15,24 @@ function drupal_ti_clear_caches() {
 	drush cr
 }
 
+#
+# Ensures that the module is linked into the Drupal code base.
+#
+function drupal_ti_ensure_module_linked() {
+	# Ensure we are in the right directory.
+	cd "$DRUPAL_TI_DRUPAL_DIR"
+
+	# This function is re-entrant.
+	if [ -L "$DRUPAL_TI_MODULES_PATH/$DRUPAL_TI_MODULE_NAME" ]
+	then
+		return
+	fi
+
+	composer config repositories.$DRUPAL_TI_MODULE_NAME path $TRAVIS_BUILD_DIR
+	composer require drupal/$DRUPAL_TI_MODULE_NAME
+	composer update -n --lock
```

Comment:
Copy paste from what we have in Commerce hack w/ composer_manager.

Seems fair enough: `-n, --no-interaction           Do not ask any interactive question`

Then we have
```
--lock                     Only updates the lock file hash to suppress warning about the lock file being out of date.
```

So that can probably be removed. In fact I doubt we even need this whole `composer update` bit and was just a double check when working with Composer Manager.